voluptuous/validators.py

```diff
@@ -33,42 +40,42 @@
     # start anchor, because fullmatch is not available in python 2.7
     "(?:"
     # dot-atom
-    r"(^[-!#$%&'*+/=?^_`{}|~0-9A-Z]+"
-    r"(\.[-!#$%&'*+/=?^_`{}|~0-9A-Z]+)*$"
+    r"(^[-!#$%&'*+/=?^_`{}|~0-9A-Z]+(\.[-!#$%&'*+/=?^_`{}|~0-9A-Z]+)*$"
```

Comment:
Why did this get joined? Can we split this again please?

Reply:
because these 2 lines can fit into one line, black merged them into:
```
    r"(^[-!#$%&'*+/=?^_`{}|~0-9A-Z]+" r"(\.[-!#$%&'*+/=?^_`{}|~0-9A-Z]+)*$"
```
which raises obvious issue "why the strings were not concatenated, which results in:
```
    r"(^[-!#$%&'*+/=?^_`{}|~0-9A-Z]+(\.[-!#$%&'*+/=?^_`{}|~0-9A-Z]+)*$"
```
The only way to keep the original version is `# fmt: on` and `# fmt: off`. I've added it, so these lines are back to being separate.